concepticondata/conceptlists.tsv

```diff
@@ -442,6 +442,7 @@ Zalizniak-2024-4583	Anna Zalizniak and Anna Smirnitskaya and Maksim Russo (Rouss
+MorenoMartinez-2012-360	Moreno-Martinez, Francisco J. and Montoro, Pedro R.	2012		360	naming test	English, Spanish	global	https://doi.org/10.1371/journal.pone.0037527.s002	MorenoMartinez2012		This work presents a new set of 360 high quality color images belonging to 23 semantic subcategories (ANIMALS, BIRDS, BODYPARTS, FLOWERS, FRUITS, INSECTS, MARINE CREATURES, NUTS, TREES, VEGETABLES, BUILDINGS, CLOTHING, DESK MATERIAL, FOOD, FURNITURE, JEWELLERY, KITCHEN UTENSILS, MUSICAL INSTRUMENT, SPORTS/GAMES, TOOLS, VEHICLES, WEAPONS, NATURE). The images were named by 230 Spanish native speakers and come with multiple psycholinguistic variables rated by the participants, listed here are mean values for: AoA (7-interval scale, 1 = 0–2 years; 7 = 13 years or more), familiarity (5-point lickert scale), typicality (5-point lickert scale), visual complexity (5-point lickert scale) and lexical frequency (obtained via the AltaVista search engine, log).		
 Migliazza-1972-714	Migliazza, Ernest	1972		714		English	Yanomamic languages		Migliazza1972		The list is part of a dissertion on the Yanomama languages. The dissertion gives information on the external situation of the language, an overview on major syntactic aspects, the phonology of the language, and a comparison between the four varieties with respect to their degree of intelligibility.	457
 Berry-1987-195	Berry, Christine and Berry, Keith	1987		195	basic	English, Indonesian	West Papuan languages	https://www.sil.org/resources/archives/37738	Berry1987		This concept list was compiled for a survey on West Papuan languages. It bears general resemblance with typical concept lists compiled for Papuan and Austronesian languages, but to our knowledge, the study does not mention a specific concept list that was used as a basic model.	62-80
+Hwang-2021-60	Hwang, Yu M. And Yoonhye, Na and Sung-Bom P.	2021		60	naming test, basic	English	Korean	https://doi.org/10.1371/journal.pone.0247118	Hwang2021		This is a newly developed categorical naming test (CNT) for stroke patients. It was created based on "The Frequency of Modern Korean Vocabulary" from the Yonsei University (1998). The final list of words (60) includes only those that are: drawable, have no more than three and no less than two syllables, have an estimated occurrence of at least 150 times per million - if they are high frequency words, or occur no more than 30 times per million - if they are low-frequency words. The final set of words was drawn as grayscale drawings and revised in an evaluation done by 10 undergraduate students. The final list includes items from two major semantic categories: living and artificial objects, 30 each.
```

Comment:
We have markdown in the website for the rendering, I suggest to follow latex practice and using `--` to indicate a dash, not single `-`. This is a minor thing, but aesthetically and orthographically important here ;-)